xc/xc7/tests/soc/litex/mini/CMakeLists.txt

```diff
@@ -1,18 +1,53 @@
-add_file_target(FILE mem.init)
-add_file_target(FILE mem_1.init)
-add_file_target(FILE mem_2.init)
+add_file_target(FILE arty_clocks.xdc)
 
-add_fpga_target(
-  NAME minilitex_arty
-  BOARD arty-full
-  SOURCES
-    minilitex_arty.v
-    VexRiscv_Lite.v
-  INPUT_IO_FILE arty.pcf
-  INPUT_XDC_FILE minilitex.xdc
-)
+function(litex_test litex_dir board device test_name)
```

Comment:
This CMake function looks a lot like the other one?  Does it make sense to make a template for this?  Maybe just provide the output source list?  Or use the method that @rw1nkler did?

Reply:
Done. In a slightly different way as the template adds the FPGA target tests directly.